docs/mp/integration/wls.adoc

```diff
@@ -0,0 +1,669 @@
+----
+
+=== References [[REST-Services-References]]
+https://helidon.io/docs/v2/#/mp/guides/02_quickstart[Helidon MP Quickstart]
+
+== JMS Integration [[JMS-Integration]]
+
+- <<JMS-Integration-Overview, Overview>>
+- <<JMS-Integration-Maven-Coordinates, Maven Coordinates>>
+- <<JMS-Integration-Usage, Usage>>
+- <<JMS-Integration-References, References>>
+
+=== Overview [[JMS-Integration-Overview]]
+
+The integration of the Oracle WebLogic Server Java Message Service (JMS) with Helidon enables Oracle WebLogic Server applications and Helidon microservices to communicate through messaging, in one or both directions.
+
+=== Maven Coordinates [[JMS-Integration-Maven-Coordinates]]
+
+To enable Oracle WebLogic Server JMS  with Helidon, add the following dependencies to your project’s `pom.xml` file:
+
+*Dependency for Reactive Messaging*
+
+[source,xml] 
+----
+<dependency>
+   <groupId>io.helidon.microprofile.messaging</groupId>
+   <artifactId>helidon-microprofile-messaging</artifactId>
+</dependency>
+----
+*Dependency for JMS connector*
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.messaging.jms</groupId>
+    <artifactId>helidon-messaging-jms</artifactId>
+</dependency>
+----
+*Dependency for Messaging Health Metrics*
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.microprofile.messaging</groupId>
+    <artifactId>helidon-microprofile-messaging-health</artifactId>
+</dependency>
+----
+
+*Dependencies for the WLS thin client JAR file*
+
+Download the Jakarta thin client JAR (`wlthint3client.jakarta`) file from https://edelivery.oracle.com/osdc/faces/Home.jspx[Oracle Software Delivery Cloud], and add it as part of the Maven dependencies:
+
+[source,xml]
+----
+<dependency>
+    <groupId>wlthint3client.jakarta</groupId>
+    <artifactId>wlthint3client-jakarta</artifactId>
+    <version>1.0</version>
+</dependency>
+----
+
+=== Usage [[JMS-Integration-Usage]]
+
+After adding the Maven dependencies, configure the Helidon JMS connector including the JMS environment properties and the JMS resources such as the connection factory, destination, and destination type.
+
+After you have configured the Helidon JMS connector, add the Java code to send and receive messages to and from Oracle WebLogic Server.
+
+Sample script to consume a message from the JMS channel:
+
+[source,java]
+----
+    @Incoming("from-wls-q")
+    public void receive(String msg) {
+        System.out.println("Process JMS message as per business logic"+msg);
+    }
+----
+
+Sample script to produce a message to the JMS channel:
+
+[source,java]
+----
+@Outgoing("to-wls-q")
+public PublisherBuilder<String> produceToJms() {
+    return ReactiveStreams.of("test1", "test2");
+}
+----
+
+For more information about this integration, see https://docs-uat.us.oracle.com/en/middleware/standalone/weblogic-server/14.1.1.0/wlshe/integrating-oracle-weblogic-server-jms-helidon.html#GUID-5E28DBC5-6CB3-49E1-B1A7-BF30191AE8AB[Integrating WebLogic JMS with Helidon 2.x]
```

Comment:
Hi @Geetha-Savithriamma 
Request you to provide reference to Helidon2 JMS Connector along with WLS Integ doc.

Helidon2 JMS Connector:
https://helidon.io/docs/v2/#/mp/reactivemessaging/05_jms